hammr/commands/image/image.py

```diff
@@ -202,16 +208,10 @@ def arg_deploy(self):
         doParser = ArgumentParser(prog=self.cmd_name + " deploy", add_help=True,
                                   description="Deploy an instance of a published image on the targeted cloud.")
         mandatory = doParser.add_argument_group("mandatory arguments")
-        mandatory.add_argument('-p', '--publish-id', dest='pid', required=True,
+        mandatory.add_argument('--file', dest='file', required=True,
+                               help="yaml/json file providing the instance parameters required for deployment on targeted cloud")
+        mandatory.add_argument('--publish-id', dest='pid', required=True,
```

Comment:
No, the -p option is already used to give hammr a password, like hammr -u root -p mypassword. As it is a short option, it can only be a single letter, so we decided to remove the short option